### DISCUSSION TITLE
Fix browser freeze-up due to unbounded column sizing calculations. #2689

### DISCUSSION THREAD

**Discussion by matthewoestreich** [material-table-core/core#7] [General]
Continued discussion per https://github.com/mbrn/material-table/pull/2689

**Comment by matthewoestreich** [comment]
@thefat32 @peacechen

>deploy a new version without this bug

[`v2.1.3`](https://www.npmjs.com/package/@material-table/core) has been published.

>configure Travis CI

It is getting late where I'm at, but I am going to set up CI/CD using GH Actions first thing tomorrow. I will keep you posted.

**Reply by peacechen** [reply]
Thanks @oze4 !

Moving forward, what are your ideas on improving this project?

**Reply by matthewoestreich** [reply]
Whatever the community agrees upon. This isn't my repo per say.

Really just want to be responsive and actively fix issues/add features. 

This repo does not exist to "rip off" mbrn, but when simple PRs take weeks to merge, something had to be done.

**Reply by peacechen** [reply]
Sounds good. To get the ball rolling, we can start creating issues in this repo. That includes bug fixes as larger architectural changes.

**Reply by matthewoestreich** [reply]
I think for now our best bet is to use discussions before creating issues, due to the fact that a lot of those issues could be solved by design changes. Thoughts?

**Reply by peacechen** [reply]
It would help to maintain two tracks: bug fixes and design changes. They could be interrelated or independent. There are times when a pressing bug fix needs to be resolved quickly. Other times design changes could fix lower priority bugs as well as previously fixed critical bugs in a more maintainable way.

Discussions are useful, but many people, especially those reporting a bug and requesting a fix, may not look at the discussion threads. I recommend creating issues for both bug fixes and design changes, then linking them to the appropriate discussion thread. That will offer visibility into the issue so that people don't create duplicate bugs as much. A discussion thread may encompass multiple issues, which is also useful in identifying what and how many individual issues it affects.

**Reply by matthewoestreich** [reply]
Done. Issues are enabled.
